format-registry/initdata/index-respec.html

```diff
@@ -92,8 +92,8 @@
   <body>
 
     <section id="abstract">
-      <p>This specification defines the <a def-id="initialization-data"></a> formats for use with the <a def-id="eme-spec"></a>.</p>
-      
+      <p>This specification defines the <a def-id="initialization-data"></a> formats for use with the <a def-id="eme-spec"></a> [[!ENCRYPTED-MEDIA]].</p>
```

Comment:
We probably don't need `<a def-id="eme-spec"></a>` anymore. You may want to add "spec" to the end of the sentence.